mev-boost-rs/src/relay_mux.rs

```diff
@@ -203,10 +203,10 @@ impl BlindedBlockProvider for RelayMux {
         let relay_indices = {
             let mut state = self.state.lock().unwrap();
             let key = bid_key_from(signed_block, &state.latest_pubkey);
-            match state.outstanding_bids.remove(&key) {
-                Some(indices) => indices,
-                None => return Err(Error::MissingOpenBid.into()),
-            }
+            state
+                .outstanding_bids
+                .remove(&key)
+                .ok_or_else(|| Error::MissingOpenBid.into())?
```

Comment:
looks like this needs a type hint but then I'm happy to merge :)

Reply:
> looks like this needs a type hint but then I'm happy to merge :)

this doesn't build, more wanting to see your preferences than contributing something more useful. Now that I type that out it seems sort of like wasting your time my apologies 😅